runtime/j9vm/j8vmi.c

```diff
@@ -204,7 +193,7 @@ JVM_CopySwapMemory(JNIEnv *env, jobject srcObj, jlong srcOffset, jobject dstObj,
 		break;
 	}
 	if (NULL != srcObj) {
-		(*env)->ReleasePrimitiveArrayCritical(env, srcObj, srcBytes, 0);
+		(*env)->ReleasePrimitiveArrayCritical(env, srcObj, srcBytes, JNI_ABORT);
```

Comment:
Why `JNI_ABORT`?

Reply:
Because the source array is not modified, there's no need to copy back to the heap (in the case where we didnt get direct access).